packages/lib/src/components/internal/SecuredFields/lib/core/handleEncryption.ts

```diff
@@ -7,7 +7,7 @@ import { SFFeedbackObj, CbObjOnFieldValid, EncryptionObj } from '../types';
 
 export function handleEncryption(pFeedbackObj: SFFeedbackObj): void {
     // EXTRACT VARS
-    const fieldType: string = pFeedbackObj.fieldType;
+    const fieldType = pFeedbackObj.fieldType;
```

Comment:
This change is not passing typechecking. Lines 14 and 17 (collapsed just below this diff) compare `fieldType` to `ENCRYPTED_EXPIRY_YEAR` and `ENCRYPTED_EXPIRY_MONTH`. However my typings have defined `fieldType` to be of type `SFFieldType`, which purposely does not include these individual month and year expiry fields, but rather only includes the parent `encryptedExpiryDate` field type. This assumption seems to hold elsewhere in the codebase, but I suppose my lack of domain knowledge here is starting to show. Maybe that's an incorrect assumption to hold, or maybe there should be additional, distinct types for the different `fieldType` properties of the various objects. Thoughts?